### PR TITLE
fix grouped clue alignment

### PR DESCRIPTION
**Pull request by desbo**
This changes the layout of clue numbers and text so that grouped clues, which have numbers like `2, 3, 4 down`, have more space and don't mess up the alignment of the clue text.

Only crosswords that have grouped clues are affected, there's no visual difference otherwise.

before
screen shot 2015-09-22 at 16 04 58

after
screen shot 2015-09-22 at 16 04 51

after (no grouped clues in this one so it looks the same as it does now)
screen shot 2015-09-22 at 16 10 49

### CODE REVIEW

**Comment by gtrufitt** [comment]
Yeah, this is much nicer! :) How's it look at this width?

image

**Comment by desbo** [comment]
@gtrufitt the space available to the numbers shrinks a little:
screen shot 2015-09-22 at 16 19 39

**Comment by gtrufitt** [comment]
:+1:

**Comment by ScottPainterGNM** [comment]
Really nice, I messaged platforms about the lack of gap between clue
numbers and the word 'across' coming out of the API and they said it was
added to their backlog but this seems a much quicker way of resolving the
issue!

On 22 September 2015 at 16:40, Gareth Trufitt <notifications@github.com>
wrote:

> [image: :+1:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/frontend/pull/10605#issuecomment-142327515>.
>



-- 
Scott Painter
QA Analyst | Dotcom

-- 

------------------------------
Visit theguardian.com. On your mobile and tablet, download the Guardian 
iPhone and Android apps theguardian.com/guardianapp and our tablet editions 
theguardian.com/editions.  Save up to 57% by subscribing to the Guardian 
and Observer - choose the papers you want and get full digital access. 
 Visit subscribe.theguardian.com

This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396

File: applications/app/crosswords/CrosswordData.scala [hunk 1]
```diff
@@ -6,15 +6,27 @@ import org.joda.time.format.ISODateTimeFormat
...
+    NumberPattern.findFirstIn(numbers).map { number =>
+      val spaced = number.split(",").mkString(", ")
+
+      DirectionPattern.findFirstIn(numbers) match {
+        case Some(direction) => spaced.concat(s" $direction")
+        case None => spaced
+      }
+    }
+  }
+
```
Comment (NathanielBennett) [inline applications/app/crosswords/CrosswordData.scala]: Nice work on the fiddly stuff - wasn't @ScottPainterGNM get CAPI to fix this, though. Mebbe worth keeping an eye out if they do ..

Comment (ScottPainterGNM) [inline applications/app/crosswords/CrosswordData.scala]: It's in their backlog - no ETA though but Chris Birchall was going to take a look for us

**Comment by Jholder112233** [comment]
What if the clue is short (just 3 words or something), how does it look?

**Comment by NathanielBennett** [comment]
@Jholder112233 that should be fine, shouldn't it? 
:+1:

File: applications/app/crosswords/CrosswordData.scala (hunk 1 above)
Comment (desbo) [inline applications/app/crosswords/CrosswordData.scala]: yeah, would definitely be better to get it from the API like that

**Comment by desbo** [comment]
@Jholder112233 like this you mean? 
screen shot 2015-09-22 at 17 17 01

**Comment by Jholder112233** [comment]
@desbo Just wanted to check :+1:

**Pull request merged**